### ISSUE TITLE
SIGUSR2 notification in log always says bandwidht limit is "disabled" even when toggling back on

### ISSUE THREAD

**Issue by drwtsn32x** [rclone/rclone#8104]
Sending SIGUSR2 to toggle the bandwidth limiter always shows "disabled" in the log, regardless of it being toggled off or back on:

```
2024/09/25 14:38:50 NOTICE: Bandwidth limit disabled by user
```

I believe this is just a reporting issue in the log.  The bandwidth limiter DOES seem to be toggled back on correctly.

According to the source, it looks like it should say "enabled" when toggling the bandwidth limiter back on (with a subsequent SIGUSR2):
https://github.com/rclone/rclone/blob/master/fs/accounting/accounting_unix.go#L36-L43

```
				tb.toggledOff = !tb.toggledOff
				tb.curr, tb.prev = tb.prev, tb.curr
				s := "disabled"
				if !tb.curr._isOff() {
					s = "enabled"
				}

				fs.Logf(nil, "Bandwidth limit %s by user", s)
```

Version info:

```
rclone v1.68.1
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.4.0-196-generic (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.23.1
- go/linking: static
- go/tags: none
```

**Comment by ncw** [comment]
This worked when I tried it

```
$ rclone rcd --bwlimit 1M -vv
2024/09/26 10:22:59 INFO  : Starting bandwidth limiter at 1Mi Byte/s
2024/09/26 10:22:59 DEBUG : rclone: Version "v1.69.0-beta.8337.b2b4f8196" starting with parameters ["rclone" "rcd" "--bwlimit" "1M" "-vv"]
2024/09/26 10:22:59 NOTICE: Serving remote control on http://127.0.0.1:5572/
2024/09/26 10:23:16 NOTICE: Bandwidth limit disabled by user
2024/09/26 10:23:19 NOTICE: Bandwidth limit enabled by user
```

And

```
kill -USR2 2141361
kill -USR2 2141361
```

Can you show an example like the one above of it failing please?

Thanks

**Comment by drwtsn32x** [comment]
Your test works for me, too.

After a lot of testing, looks like I can only trigger the problem when I use different upload/download bandwidth limits and the download one is "off", like this:

```
$ rclone rcd --bwlimit 1M:off -vv
2024/09/26 08:10:08 INFO  : Starting bandwidth limiter at 1Mi:off Byte/s
2024/09/26 08:10:08 DEBUG : rclone: Version "v1.68.1" starting with parameters ["rclone" "rcd" "--bwlimit" "1M:off" "-vv"]
2024/09/26 08:10:08 NOTICE: Serving remote control on http://127.0.0.1:5572/
2024/09/26 08:10:17 NOTICE: Bandwidth limit disabled by user
2024/09/26 08:10:18 NOTICE: Bandwidth limit disabled by user
```

```
$ kill -USR2 716120
$ kill -USR2 716120
```

Thank you

**Comment by ncw** [comment]
Excellent investigations thank you. I can see that is a left over bug from when we went from 1 core bandwidth limiters to 3 (rx, core and tx).

Please give this a go. I also wrote the bandwidth in the log as that was requested on the forum!

[v1.69.0-beta.8341.19458e845.fix-8104-bwlimit-logs](https://beta.rclone.org/branch/fix-8104-bwlimit-logs/v1.69.0-beta.8341.19458e845.fix-8104-bwlimit-logs/) on branch [fix-8104-bwlimit-logs](https://github.com/rclone/rclone/tree/fix-8104-bwlimit-logs) (uploaded in 15-30 mins)

**Comment by drwtsn32x** [comment]
That was quick!  Initial tests look like this has been resolved. Appreciate the improved logging, too!

**Comment by ncw** [comment]
Thank you for testing.

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.69 and in v1.68.2 if we end up making that.
